patcher/src/widgets/Controller/components/PatcherError/index.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import * as React from 'react';
+import styled from 'react-emotion';
+import { PatcherError } from '../../../../services/patcher';
+
+const Alert = styled('div')`
+  overflow: hidden;
+  background-color: orange;
+  color: black;
+  padding: 3px 15px;
+  position: relative;
+  pointer-events: bounding-box;
```

Comment:
What does this do? I've never seen this before

Reply:
Errr this isn't actually a request for change, was just asking a question.

Reply:
Looks like its a chrome thing, and really meant for svg.  Should probably change to : all;